### ISSUE TITLE
Bridge Riot---Slack: not all Riot messages are posted to Slack

### ISSUE THREAD

**Issue by maudnals** [vector-im/riot-web#11096]
### Description

Slack bridge: not all messages posted from Riot are visible in Slack. For the **same user** posting from Riot, their messages are sometimes visible in Slack and sometimes not.
I would expect their messages to always be visible on Slack.

### Steps to reproduce

- Create a new Riot room
- Create a new Slack channel 
- Bridge them using the Events API
- For User X posting from Riot, their messages are sometimes visible in Slack and sometimes not. 

Logs being sent: no

### Version information

- **Platform**: web + desktop

Riot web:
- **Browser**: Chrome
- **OS**: macOS
- **URL**: https://riot.im/app/#/room/#kilt-general:matrix.org

Slack desktop: macOS, version 4.0.1

**Comment by jryans** [comment]
@Half-Shot, would this be better to track on the [bridge repo](https://github.com/matrix-org/matrix-appservice-slack)?

**Comment by Half-Shot** [comment]
Yep, but I'm also unable to reproduce or find issues with the user's setup. If we find a bug, then I'm happy to transfer it.

@maudnals can you provide an eventId for messages which didn't make it across, and confirm if this is still happening?

**Comment by maudnals** [comment]
@Half-Shot Thanks for looking into this 👍 
 
> Can you provide an eventId for messages which didn't make it across
Sure, here it is:
```json
  "origin_server_ts": 1570088242341,
  "type": "m.room.message",
  "event_id": "$OxXN0nESTQxe2vyjpIv7tr49ycJ-FEsGvTr3nTfzvxs",
  "room_id": "!qgncBLmNTYJRPSfSHZ:matrix.org"
```
> Confirm if this is still happening
Can't answer this; the user has not been writing in the riot channel again.
